src/MoBi.Core/Domain/Model/MoBiProject.cs

```diff
@@ -147,5 +145,14 @@ public IEnumerable<IObjectBase> All()
       {
          return All<IObjectBase>().Union(Simulations);
       }
+
+      /// <summary>
+      /// Returns a list of simulations that have a module where the name matches <paramref name="module"/>
+      /// This indicates that the <paramref name="module"/> was used as a template for the simulation
+      /// </summary>
+      public IReadOnlyList<IMoBiSimulation> SimulationsCreatedUsing(Module module)
```

Comment:
SimulationUsing? 
Why created?

Reply:
That just matches the existing overload, but they can both be renamed